generator/overrides/gen.go

```diff
@@ -0,0 +1,435 @@
+	if typeToOverride.Fields != nil {
+		for _, field := range typeToOverride.Fields {
+			fieldPos := field.RawField.Pos()
+			if !fieldPos.IsValid() {
+				errors = append(errors,
+					fmt.Errorf("Field %v in type %v doesn't have a valid position in the source file",
+						field.Name,
+						typeToOverride.Name,
+					))
+				continue
+			}
+			overridesMarker := FieldOverridesInclude{}
+			if markerEntry := field.Markers.Get(overridesFieldMarker.Name); markerEntry != nil {
+				overridesMarker = markerEntry.(FieldOverridesInclude)
+			}
+			fieldChanges[fieldPos] = fieldChange{
+				FieldInfo:      field,
+				OverrideMarker: overridesMarker,
+			}
+		}
+	}
+
+	overrideGenDecl := astcopy.GenDecl(typeToOverride.RawDecl)
+	if typeToOverride.Markers.Get(overridesTypeMarker.Name) != nil {
+		overrideGenDecl.Doc = updateComments(overrideGenDecl, overrideGenDecl.Doc, `.*`, ` *\+`+overridesTypeMarker.Name)
+	}
+	if newTypeToProcess.DropEnumAnnotation {
+		overrideGenDecl.Doc = updateComments(
+			overrideGenDecl, overrideGenDecl.Doc,
+			`.*`,
+			` *`+regexp.QuoteMeta("+kubebuilder:validation:Enum=")+`.*`,
+		)
+	}
+
+	overrideGenDecl.Doc = updateComments(
+		overrideGenDecl, overrideGenDecl.Doc,
+		`.*`,
+		` *`+regexp.QuoteMeta("+devfile:jsonschema:generate"),
+	)
+
+	if newTypeToProcess == g.rootTypeToProcess {
+		overrideGenDecl.Doc = updateComments(
+			overrideGenDecl, overrideGenDecl.Doc,
+			`.*`,
+			``,
+			"+devfile:jsonschema:generate",
+		)
+	}
+
+	moreTypesToAdd := []typeToProcess{}
+	overrideGenDecl = astutil.Apply(overrideGenDecl,
+		func(cursor *astutil.Cursor) bool {
+			if typeSpec, isTypeSpec := cursor.Node().(*ast.TypeSpec); isTypeSpec {
+				if alreadyFoundType != nil {
+					errors = append(errors,
+						fmt.Errorf("types %v and %v are defined in the same type definition - please avoid defining several types in the same type definition",
+							alreadyFoundType.Name,
+							typeSpec.Name,
+						))
+					return false
+				}
+				typeSpec.Name.Name = newTypeToProcess.OverrideTypeName
+			}
+			if astField, isField := cursor.Node().(*ast.Field); isField {
+				if newTypeToProcess == g.rootTypeToProcess &&
+					cursor.Index() == 0 {
+					cursor.InsertBefore(&ast.Field{
+						Type: &ast.Ident{Name: "OverridesBase"},
+						Tag:  &ast.BasicLit{Kind: token.STRING, Value: "`json:\",inline\"`"},
+					})
+				}
+
+				fieldChange := fieldChanges[cursor.Node().Pos()]
+				field := fieldChange.FieldInfo
+
+				overridesMarker := fieldChange.OverrideMarker
+
+				shouldSkip := func(overridesMarker FieldOverridesInclude) bool {
+					if overridesMarker.Omit ||
+						(overridesMarker.OmitInPlugin && g.IsForPluginOverrides) {
+						return true
+					}
+					return false
+				}
+
+				if shouldSkip(overridesMarker) {
+					cursor.Delete()
+					return true
+				}
+
+				if overridesMarker.Description != "" {
+					astField.Doc = updateComments(
+						astField, astField.Doc,
+						` *\+[^ ][^ ]*.*`,
```

Comment:
```suggestion
						` *\+[^ ]+.*`,
```

Reply:
Fixed in commit https://github.com/devfile/api/pull/133/commits/37c1e341bb7bb541566dbf0b63a536b64e1b6028